### ISSUE TITLE
[FEATURE]  Implement Single Logout Service

### ISSUE THREAD

**Issue by micotodev** [christopherpickering/remix-auth-saml#22]
## Feature Request

Thanks for this package, has been really helpful. Had a look through regarding the Single Logout Service, and looks unimplemented yet?

I assume this would need to parse the SP, check if signatures are signed and then delete the cookie for the requested user?

I imagine we just need to implement against this [samlify test](https://github.com/tngan/samlify/blob/1d7876358176942bc1ea31a71ec97369f29a6c6a/test/flow.ts#L503)?

**Comment by christopherpickering** [comment]
Yep, thats right. If you have already started, you can send a pr, otherwise I'll give it a shot sometime 👍🏽

**Comment by micotodev** [comment]
No problem, need to implement in a project first then I'll try and push a PR back here. 

Issue I am currently looking at which is ok in userland but hard to scope for this package.

The user would have to supply a callback function that can both find the session of the user and remove it. Think this one might be best left generic with a docs example as there are far too many scenarios to cover e.g some just store the user id, some may store the entire user object.
